src/Reveal.Sdk.Dom/Data/TableDataSourceItem.cs

```diff
@@ -15,5 +15,12 @@ public string Table
             get => Properties.GetValue<string>("Table");
             set => Properties.SetItem("Table", value);
         }
+
+        [JsonIgnore]
+        public string CustomQuery
```

Comment:
do not add any custom query properties to these objects. It will not be used by anything

Reply:
Updated